api/application/client.go

```diff
@@ -629,7 +629,7 @@ func (c *Client) DestroyConsumedApplication(in DestroyConsumedApplicationParams)
 
 	if apiVersion > 9 {
 		if in.MaxWait != nil && !in.Force {
-			return nil, errors.New("--force is required when --max-wait is provided")
+			return nil, errors.New("--force is required when --no-wait is provided")
```

Comment:
I think this should be max-wait as no-wait is a cli layer term.

Reply:
Makes sense, I'll revert this change